lib/rubocop/cop/mavenlint/use_application_controller.rb

```diff
@@ -0,0 +1,27 @@
+require 'rubocop'
+
+module RuboCop
+  module Cop
+    module Mavenlint
+      # Enforce controllers inheriting from ApplicationController instead of ActionController::Base
+      #
+      # For example
+      #
+      #   class SomeController < ActionController::Base
+      #   end
+      #
+      # should be
+      #
+      #   class SomeController < ApplicationController
+      #   end
+      #
+      class UseApplicationController < RuboCop::Cop::Cop
```

Comment:
CRITICAL: Do we need to enforce this linting rule in rubocop.yml?

Reply:
Thanks for catching that, its added now!